six/common/builtins/_util.h

```diff
@@ -7,7 +7,10 @@
 #include <Python.h>
 #include <six_types.h>
 
+#define _DOT "."
```

Comment:
Do we really need to alias the dot ?

Reply:
We don't. I felt it reads a little better than:
```FOO_MODULE "." BAR_FUNCTION```
```FOO_MODULE _DOT BAR_FUNCTION```

I can definitely remove it you prefer

Reply:
It doesn't really matters, I prefer the `"."` version but feel free to merge in any case ;)